libs/labelbox/tests/data/annotation_import/test_conversation_import.py

```diff
@@ -1,45 +0,0 @@
-import uuid
```

Comment:
Deleted this file since we are testing already for conversation entity import with test_generic_data_types